src/plugins/nest.js

```diff
@@ -0,0 +1,293 @@
+'use strict'
+
+function createWrapCreate (tracer, config) {
+  return function wrapCreate (create) {
+    return function createWithTrace () {
+      const instance = arguments[0]
+      const methodName = arguments[2]
+
+      const handler = create.apply(this, arguments)
+      return function (req, res, next) {
+        let opName = 'nest.request'
+        if (instance.constructor && instance.constructor.name) {
+          opName = instance.constructor.name
+        }
+        const scope = tracer.scope()
+        const childOf = scope.active()
+        const span = tracer.startSpan(`${opName}.${methodName}`, {
+          childOf,
+          tags: {
+            'component': 'nest',
+            'nest.request.method': req.method,
+            'nest.request.path': req.route.path
```

Comment:
Can we define these keys as package level constance ?

Reply:
BEST AUTOCORRECTION EVER! :)